cypress/integration/404/404_spec.js

```diff
@@ -14,8 +14,9 @@ describe('Testing the 404 page:', function () {
   });
 
   it('/search page should load when the search button is clicked', function () {
-    cy.get('[data-automation-id="404-search-button"]').as('404SearchButton');
-    cy.get('@404SearchButton').click();
+      cy.wait(5000);
```

Comment:
I'm not a huge fan of hard coded waits unless it's completely unavoidable. I would wonder if the right thing to do is do a timeout that waits for something to be on the page and then continue. 

Something like this:
https://stackoverflow.com/questions/16149431/make-function-wait-until-element-exists/47776379 (look at green checked solution)